daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -97,14 +97,22 @@ cmdTest numProcessors =
        progDesc "Test the given DAML file by running all test declarations."
     <> fullDesc
   where
-    cmd = execTest
+    cmd = runTestsInProjectOrFiles
       <$> many inputFileOpt
       <*> fmap UseColor colorOutput
       <*> junitOutput
       <*> optionsParser numProcessors optPackageName
     junitOutput = optional $ strOption $ long "junit" <> metavar "FILENAME" <> help "Filename of JUnit output file"
     colorOutput = switch $ long "color" <> help "Colored test results"
 
+runTestsInProjectOrFiles :: [FilePath] -> UseColor -> Maybe FilePath -> Compiler.Options -> IO ()
+runTestsInProjectOrFiles [] color mbJUnitOutput cliOptions = do
+    project <- readProjectConfig $ ProjectPath "."
```

Comment:
This needs to be changed to use `getProjectPath`. We should also handle the case where `getProjectPath` returns `Nothing`, i.e., we are outside of a project, and fallback to running `execTest` with an empty list of files. (That should fix your CI failures).